DANS-KNAW/easy-mirror-deposit
# DD-1657: simplify - keep only writing writing RDA bags to temporary vault

Fixes DD-1657: simplify - keep only writing writing RDA bags to temporary vault

# Description of changes

# How to test

* In dans-core-systems: start VM deasy
* on a fresh `deasy-2022-06-21.box`
  * `/etc/yum.repos.d/dans.knaw.nl.repo` should contain 

        [local]
        name=Local Repository
        baseurl=file:///vagrant/.dans-rpms

  * create inboxes as specified in `/etc/opt/dans.knaw.nl/easy-mirror-deposit/config.yml`
* In dans-core-systems: `deploy.py -m easy-mirror-deposit deasy`
* on deasy again:

      sudo rm /etc/systemd/system/easy-mirror-deposit.service.d/debug.conf
      sudo systemctl daemon-reload
      sudo systemctl restart easy-mirror-deposit
      echo > /var/opt/dans.knaw.nl/tmp/transfer-inboxes/archaeology/doi-10-5072-fk2-xcfq1bv1.1.zip

   * this creates `/data/easy-mirror-store/xc/fq/doi-10-5072-fk2-xcfq1bv1.1.zip`


# Related PRs

(Add links)

*

# Notify

@DANS-KNAW/core-systems
